README.md

```diff
@@ -118,7 +118,18 @@ Collection of examples that solve specific problems using small pieces of code.
 
 ## Contributing
 
-Contributions are welcome!
+- Contributions make the open source community such an amazing place to learn, inspire, and create.
+- Any contributions you make are **truly appreciated**.
+- Check out our [contribution guidelines](/CONTRIBUTING.md) for more information.
+
+<h2>
+License
+</h2>
```

Comment:
I think it will be better if we use markdown like tags for h2

```md
## License
```

Reply:
done